Security/Http/Firewall/LegacyTwoFactorListener.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\TwoFactorBundle\Security\Http\Firewall;
+
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+
+class LegacyTwoFactorListener extends TwoFactorListener
```

Comment:
1) This has to implement `Symfony\Component\Security\Http\Firewall\ListenerInterface`, otherwise it will break with Symfony <4.3 because these older versions are still type-hinting `ListenerInterface`.
2) I'd suggest to use the decorator pattern for this class instead of extending `TwoFactorListener`. So basically calling `$this->decoratedListener->__invoke()` in `handle()`. Test case can then focus entirely on the new implementation.

Reply:
Good point, I agree. I update everything as soon as I can

Reply:
Good point, I agree. I update everything as soon as I can